run_web.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# wait for MYSQL server to start
+
+until netcat -z -v -w30 db 3306
```

Comment:
Can we make this work different, like, not require a port or something? Or grab the port from an ENV var?